### PR TITLE
Show user's Wii number in DisplayError()

### PR DESCRIPTION
**Pull request by Naim2000**

### CODE REVIEW

File: patcher.cpp [hunk 1]
```diff
@@ -11,7 +11,8 @@ static int DisplayError(NWC24Config& config, std::string_view message, s32 error
 
   std::cout << message << std::endl;
   printf("Error Code: %d\n", error_code);
```
Comment (Naim2000) [inline patcher.cpp:13]: Half curious about why printf is here

**Comment by Naim2000** [comment]
image

File: patcher.cpp (hunk 1 above)
Comment (noahpistilli) [reply patcher.cpp:13]: Relic from C.

**Comment by noahpistilli** [comment]
Later I do want to have a map of common error codes and what they mean.

File: patcher.cpp [hunk 2]
```diff
@@ -11,7 +11,8 @@ static int DisplayError(NWC24Config& config, std::string_view message, s32 error
...
-  std::cout << "Please doing the WiiLink Discord for support." << std::endl;
+  std::cout << "Wii number: " << config.GetFriendCode() << std::endl << std::endl;
```
Comment (NecroNovaVGitHub) [inline patcher.cpp:14]: Recommendation: make "number" into "Number" (aka add slight capitalization). Looks a little weird with "Wii" only being capitalized.

**Pull request merged**